pptx_blueprint/__init__.py

```diff
@@ -85,8 +90,38 @@ def _find_shapes_in_slide(slide):
     def save(self, filename: _Pathlike) -> None:
         """Saves the updated pptx to the specified filepath.
 
-        Args:
+         Args:
             filename (path-like): file name or path
         """
         # TODO: make sure that the user does not override the self._template_path
         pass
+
+    def save_pdf(self, file_path: _Pathlike) -> None:
+        """Exports the updated pptx to the specified filepath as pdf file.
+
+        Args:
+            file_path (path-like) file name or path
+        """
+
+        try:
+            subprocess.run(['libreoffice', '--version'],
+                           stdout=subprocess.DEVNULL)  # check if libreoffice is installed
+
+            path = Path(file_path)
+            outdir = path.parent
+            file_name = path.name
+
+            # create temporary  directory
+            with tempfile.TemporaryDirectory() as tmpdirname:
+                template_temporary_path = f'{tmpdirname}/{file_name}'
+
+                # save current Template as pptx in temporary directory
+                # TODO replace with self.save() method
+                self._presentation.save(template_temporary_path)
+
+                export_cmd = ['libreoffice', '--headless', '--convert-to',
```

Comment:
I'm not a huge fan of defining these kind of commands in the middle of the other code, because this yet again hinders my reading flow. I would prefer to have a `LibreOfficeWrapper` class that encapsulates both checking that LibreOffice is present as well as converting a file to a PDF. This would also facilitate to write unit tests, because you could mock this class.